service/actions/preferences.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import Client from 'service/client';
+
+import {PreferencesTypes} from 'service/constants';
+import {bindClientFunc, forceLogoutIfNecessary, requestData, requestFailure} from './helpers';
```

Comment:
requestData and requestFailure are actually not necessary here, but we can fix that later